tests/test_permissions.py

```diff
@@ -40,17 +39,6 @@ def test_if_valid_api_key_custom_header_then_permission_granted(
         assert response.status_code == 200
 
 
-@pytest.mark.parametrize("hasher", PASSWORD_HASHERS)
```

Comment:
This is a leftover from #244 which I didn't see before.

We don't need this test anymore now that we always use SHA512.